src/main/drivers/vtx_common.c

```diff
@@ -174,8 +169,11 @@ const char *vtxCommonLookupChannelName(const vtxDevice_t *vtxDevice, int channel
 }
 
 //Converts frequency (in MHz) to band and channel values.
-bool vtxCommonLookupBandChan(const vtxDevice_t *vtxDevice, uint16_t freq, uint8_t *pBand, uint8_t *pChannel)
+//If frequency not found in the vtxtable then band and channel will return 0
```

Comment:
Not sure that this statement is actually true with the existing code?

Reply:
I'm confused. I added that comment because the function used to be boolean and that determined whether a frequency was found. Now the band and channel will be 0.
```
//Converts frequency (in MHz) to band and channel values.
//If frequency not found in the vtxtable then band and channel will return 0
void vtxCommonLookupBandChan(const vtxDevice_t *vtxDevice, uint16_t freq, uint8_t *pBand, uint8_t *pChannel)
{
    *pBand = 0;
    *pChannel = 0;
    if (vtxDevice) {
        // Use reverse lookup order so that 5880Mhz
        // get Raceband 7 instead of Fatshark 8.
        for (int band = vtxTableBandCount - 1 ; band >= 0 ; band--) {
            for (int channel = 0 ; channel < vtxTableChannelCount ; channel++) {
                if (vtxTableFrequency[band][channel] == freq) {
                    *pBand = band + 1;
                    *pChannel = channel + 1;
                    return;
                }
            }
        }
    }
}
```

Reply:
Turns out I misread the code after reading the comment - please disregard.